dll.py

```diff
@@ -1,52 +1,163 @@
+"""
+Module for Doubly Linked Lists
+"""
+
+
 class Node(object):
 
-    def __init__(self, vertex: "Vertex", prev: "Vertex"=None, next: "Vertex"=None):
+    def __init__(self, vertex: "Vertex", prev: "Vertex" = None, next: "Vertex" = None):
         self._vertex = vertex
-        self._prev = prev
-        self._next = next
+        self.prev = prev
```

Comment:
Ik had dacht ik precies van jou of Claudia het commentaar gekregen dat het met zo'n _ moest. Dus waarom draai je dat hier dan weer terug? En waarom zou je een `@property` maken van value? Want daarmee krijg je er ook getters en setters bij en dan hebben we misschien de `__init__` niet nodig :)

Reply:
Ja sorry. Ik had er eerst ook properties van gemaakt, maar dan mocht ik het niet aanpassen (in bv push/pop/append). Daarom had ik hier weer public variabelen van gemaakt (en ter conventie de _ weer weggehaald).